migrations/20200619201036-create-image.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("images", {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      public_Id: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      imageURL: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      postId: {
+        type: Sequelize.INTEGER,
+        references: {
+          model: "posts",
+          key: "id",
+        },
+        onUpdate: "CASCADE",
+        onDelete: "CASCADE",
```

Comment:
`onDelete: CASCADE` is only for many-to-many relations, if not we use `SET NULL`